Source/WebCore/platform/graphics/avfoundation/AVTrackPrivateAVFObjCImpl.mm

```diff
@@ -101,8 +101,8 @@ - (id)optionID;
     if (!m_assetTrack)
         return;
 
-    [m_assetTrack loadValuesAsynchronouslyForKeys:assetTrackConfigurationKeyNames() completionHandler:[weakThis = WeakPtr(this)] () mutable {
-        callOnMainThread([weakThis = WTFMove(weakThis)] {
+    [m_assetTrack loadValuesAsynchronouslyForKeys:assetTrackConfigurationKeyNames() completionHandler:[weakThis = ThreadSafeWeakPtr { *this }] {
+        callOnMainThread([weakThis] {
```

Comment:
Why not keeping moving it? This would reduce count churning and it seems the callback is a completion handler so would be called only once.